admin_guide/metrics-sizing-guidelines.adoc

```diff
@@ -0,0 +1,164 @@
+= Openshift Metrics Sizing Guidelines
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Openshift Metrics Basics
+
+Monitoring openshift pods is important task from both, Openshift administrator and Openshift user point
+of view. Openshift administrators need a way to monitor pods of interest and see how they perform
+ and react if necessary to fix critical issue, at other side Openshift users also need simple and
+ comprehensive overview of pods where their application run.
+
+This guide will provide Openshift metrics guidlines which Openshift administrators and users can
```

Comment:
will provide capacity planning guidelines for administrators when sizing an environment to host OpenShift metrics.